tests/lowering/multiple_rule_clauses.eclair

```diff
@@ -0,0 +1,231 @@
+// RUN: split-file %s %t
+
+// RUN: %eclair compile --emit ra %t/program.eclair > %t/actual_ra.out
+// RUN: diff %t/expected_ra.out %t/actual_ra.out
+
+// RUN: %eclair compile --emit eir %t/program.eclair > %t/actual_eir.out
+// RUN: diff %t/expected_eir.out %t/actual_eir.out
+
```

Comment:
@luc-tielen We need to add the following line, which seems to have been missed here:
```bash
// RUN: %eclair compile --emit llvm %t/program.eclair > %t/actual_llvm.out
```
Once this line is added, the integration tests all pass !

Reply:
Oops, forgot this line! I must've forgotten tis somehow :thinking:.. thanks!